lib/libzfs/libzfs_sendrecv.c

```diff
@@ -4150,6 +4150,12 @@ zfs_setup_cmdline_props(libzfs_handle_t *hdl, zfs_type_t type,
 			 */
 			if (!zfs_prop_valid_for_type(prop, type, B_FALSE) &&
 			    !zfs_prop_user(name)) {
+				if (type == ZFS_TYPE_VOLUME &&
+				    strncmp("mountpoint", name, 10) == 0) {
```

Comment:
```suggestion
				    strcmp("mountpoint", name) == 0) {
```

Reply:
We want to exactly match the property name "mountpoint", not any property with "mountpoint" as a prefix.